.github/workflows/dry-run-publish.yaml

```diff
@@ -22,5 +22,12 @@ jobs:
       - name: Install & Build Dependencies
         run: pnpm bootstrap
 
+      - name: Stage and commit build artifacts
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
```

Comment:
Can you set GitHub user using this (new) action: https://github.com/swisspost/design-system/blob/main/.github/actions/configure-github-user/action.yaml

You can update the action if you need, adding inputs for example.